Sender/BatteryDataStreamingTests/BatteryDataStreamingTests.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using Xunit;
+using BatteryDataStreaming;
+
+namespace BatteryDataStreamingTests
+{
+    public class BatteryDataStreamingTests

```

Comment:
Tests for the output format are missing. Focus on testing your functionality - the most crucial is the output format.
In the absence of a test for the output format, any future mistakes will be caught only while integrating with the receiver.